pkgs/development/python-modules/python-hcl2/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  lark,
+  setuptools_scm,
+}:
+
+buildPythonPackage rec {
+  pname = "python-hcl2";
+  version = "4.3.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-7xubrQGLz8H+J5IESXQpnlWRRf6W48opjB5elQDI3mY=";
+  };
+
+  disabled = pythonOlder "3.7";
+
+  build-system = [ setuptools_scm ];
```

```suggestion
  build-system = [
    setuptools
    setuptools-scm
  ];
```
Could you add `setuptools` explicitly?
Also, we adopt `-` instead of `_` in nixpkgs, so `setuptools_scm` should be `setuptools-scm`.